blog/blog_gsoc_2022.md

```diff
@@ -0,0 +1,63 @@
+(sprint_testimonial)=
+# Meet our 2022 PyMC Google Summer of Code Students
+
+:::{post} June 24, 2022
+:tags: gsoc
+:category: development
+:author: Reshama Shaikh, GSOC 2022 Folks
+:image: 0
+:exclude:
+
+We are excited to introduce to the community our cohort of GSoC students working on PyMC, Aesara and ArViZ.
+:::
+
+## Purna Chandra Mansingh
+
+**Project Name**
+
+Increase Support for Batched Multivariate Distributions
+
```

Comment:
```suggestion
**Project Name**

Increase Support for Batched Multivariate Distributions

**Mentors**
- [Sayam Kumar](https://github.com/Sayam753)
- [Ricardo Vieira](https://github.com/ricardoV94)
```

Reply:
@OriolAbril What do you think of adding the Mentors names? 
I think it would be nice to add their name(s).